### ISSUE TITLE
Game Request: [light] Dies irae ~Interview with Kaziklu Bey~

### ISSUE THREAD

**Issue by YoxalLoyal** [morkt/GARbro#112]
https://vndb.org/v18685
Developer: light
Engine: Malie
File Extension: .dat

Hello. Do you know if your tool supports light's Dies irae ~ Interview with Kaziklu Bey ~? I tried it and it just says the data.dat file could not be opened as a resource archive. I am using the latest version. Maybe I need to download and put the source code that supports 'malie' somewhere in the directory? Sorry, I am new to using your tool. Thanks in advance for your assistance.
![screenshot 1156](https://user-images.githubusercontent.com/34036620/33279767-c12a9a0e-d36d-11e7-8df2-8b268910d857.png)

**Comment by morkt** [comment]
i've updated formats database, you may download it using "check for updates" menu in recent pre-release.

**Comment by YoxalLoyal** [comment]
Thank you for your reply. I tried again with your pre-release version, but it still does not work for me. Maybe I'm doing something wrong. Does the GARbro folder have to be in the same directory as the game files or can it be somewhere else like in DESKTOP? Also, where do you put the source codes for Malie? I think I need that to successfully open and extract the files. Sorry for the trouble and thanks again.

**Comment by morkt** [comment]
like i said, choose "help | check for updates" in application menu to download updated formats database.

**Comment by YoxalLoyal** [comment]
Hello again. I have another question since being able to use your tool. Do you know any tool that converts .webp into .png files? I have been using an online converter to convert them, but it gives me a low quality .jpg-like png image and I'm not sure if that is the file itself or if the converter is not good. Thanks in advance.

**Comment by morkt** [comment]
ummm... GARbro? select files -> Convert multimedia [F6 button]

**Comment by YoxalLoyal** [comment]
I previously tried to convert the files upon extraction with GARbro first but the program does not seem to support webp files. Or it doesn't recognize webp files as an image file.

**Comment by morkt** [comment]
guess something wrong with your GARbro setup again, because webp images are fully supported and have been for quite some time.

**Comment by YoxalLoyal** [comment]
So my PC decided to block some of the files from both GARbro and the Source Code folders. I previously only unblocked all the files from the GARbro-v1.4.32.1978 folder to make the extraction work. 

Today, from the GARbro-1.4.33-pre1 folder, I only unblocked every file in the ArcFormats folder as well as the ones in Malie and WebP folders. I also unblocked the individual files in the GARbro-1.4.33-pre1 and Image.Convert folders. Is there anything else I need to unblock or move to make the image conversion work? Apologies for the inconvenience.

**Comment by morkt** [comment]
you've probably missed the most crucial part that is responsible for WEBP decoding, `libwebp.dll` in x86/x64 folders. but rather than clicking DLLs one by one, it's much easier to unblock ZIP archive you downloaded and then extract it, or use third-party tool for extraction, like Winrar or 7zip.

**Comment by YoxalLoyal** [comment]
Hmm, I did everything you have told me and it still does not convert the .webp files to .png format. I have all the prerequisites for your program so I'm not exactly sure what is wrong. I even tried updating my .NET, but it seems like I already have the newest version. Everything is now unblocked too. When I try to convert the webp files, GARbro says no media file is selected when I try to convert a random webp file. I tried everything like moving the stuff from GARbro-1.4.33-pre1 to GARbro-v1.4.32.1978, though I'm pretty everything from that folder is already in the main GARbro folder so that really didn't do much.

I probably won't bother you much on this anymore and just go back to what I've been doing till now. Thanks for all the help btw. Maybe I will have better luck with an earlier version of GARbro? I'll try doing that later.

Here are three screenshots of the problems I've been having:
![screenshot 382](https://user-images.githubusercontent.com/34036620/33893810-624d9f36-df2a-11e7-88ba-66bfeb300746.png)
![screenshot 383](https://user-images.githubusercontent.com/34036620/33893811-6261f5c6-df2a-11e7-9798-17215db6f979.png)
![screenshot 384](https://user-images.githubusercontent.com/34036620/33893812-627656b0-df2a-11e7-93a9-5e53dc6c8acd.png)
